src/radix_connect/mobile/relay_service/models/request/request_method.rs

```diff
@@ -0,0 +1,43 @@
+use crate::prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub enum Method {
+    SendRequest,
```

Comment:
please add at least a oneline documentation to all types and to all variants :)

Reply:
Will link the docs of RadixConnectRelay